core/buildTokens.js

```diff
@@ -0,0 +1,147 @@
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const StyleDictionary = require('style-dictionary');
+
+const { fileHeader } = StyleDictionary.formatHelpers;
+
+// Empty for as an example of how we can add some extra variables, not from the tokens JSON
+const customVariables = ``;
+
+// CSS vanilla :root format
+StyleDictionary.registerFormat({
+  name: 'rootFormat',
+  formatter({ dictionary, file }) {
+    // Add a prefix to all variable names
+    const prefixedVariables = dictionary.allProperties.map((prop) => {
+      return `  --${prop.name}: ${prop.value};`;
+    });
+
+    return (
+      fileHeader({ file }) +
```

Comment:
Can we insert a comment block in the file contents that get generated that inform a user opening the file that it is auto-generated and they should not make changes in that file?

Reply:
Yes, its in fact the default behaviour 😂, I will adjust that!

Reply:
Fixed!